week2/2-website/index.html

```diff
@@ -16,6 +16,63 @@
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
-
+    <header>
+        <div class="content-container">
+            <div class="header-layout">
+                <img class="logo" src="img/karma-logo.svg" alt="Karma logo">
+                <nav class="menu">
+                    <ul>
+                        <li class="selected">Meet Karma</li>
+                        <li>How it Works</li>
+                        <li>Store</li>
+                        <li>Blog</li>
+                        <li>Help</li>
+                        <li>Login</li>
+                    </ul>
+                </nav>
+            </div>
+        </div>
+    </header>
+     <div id="container">
+         <div class="tag-container">
+             <div class="content-container tag-content">
+                <h1 id="title">Introducing Karma</h1>
+                <h3 id="tagline">Bring WiFi with you, everywhere you go.</h3>
+                <button>Learrn More</button>
+             </div> 
+         </div>
+        <main id=benefits>
```

Comment:
```suggestion
        <main id="benefits">
```
missing double quotations :)